engine/lib/dependencies.ml

```diff
@@ -527,7 +527,21 @@ module Make (F : Features.T) = struct
             when List.for_all variants ~f:(fun variant -> variant.is_record)
                  && Concrete_ident.is_constructor from_id ->
               None
+          (* We don't need aliases for fields of types. *)
+          | Type _
+            when match List.last (Concrete_ident.to_view from_id).rel_path with
+                 | Some (`Field _) -> true
+                 | _ -> false ->
+              None
+          (* We don't need aliases for methods of trait impls. *)
+          | Impl _
+            when match List.last (Concrete_ident.to_view from_id).rel_path with
+                 | Some (`AssociatedItem _) -> true
+                 | _ -> false ->
+              None
```

Comment:
Can you pull out `List.last (Concrete_ident.to_view from_id).rel_path` in a lambda in a binding above, say `inspect_view`, and do `[%matches? Some (`Field _)] (inspect_view ())` or something?